include/verilated.cpp

```diff
@@ -1459,11 +1459,13 @@ void VL_SFORMAT_X(int obits, void* destp, const char* formatp, ...) VL_MT_SAFE {
 
 void VL_SFORMAT_X(int obits_ignored, std::string& output, const char* formatp, ...) VL_MT_SAFE {
     if (obits_ignored) {}
-    output = "";
+    std::string temp_output;
+    temp_output = "";
```

Comment:
```suggestion
```
Strings init to empty so this is not needed.  (You can just apply this suggestion in github directly, you don't have to push it.)

Reply:
Done! I've never used the commit suggestion thing before. Very handy